APM E2E TestUpdateConfiguration regression

E2E test TestUpdateConfiguration fails.

```
[2019-07-13T15:19:32.382Z]     --- FAIL: TestUpdateConfiguration/Check_the_value_of_a_parameter_in_the_configuration (0.11s)
[2019-07-13T15:19:32.382Z]         require.go:157: 
[2019-07-13T15:19:32.382Z]             	Error Trace:	configuration_test.go:93
[2019-07-13T15:19:32.382Z]             	Error:      	Not equal: 
[2019-07-13T15:19:32.382Z]             	            	expected: 0
[2019-07-13T15:19:32.382Z]             	            	actual  : 5
[2019-07-13T15:19:32.382Z]             	Test:       	TestUpdateConfiguration/Check_the_value_of_a_parameter_in_the_configuration
```

It expects that `config.Output.Elasticsearch.CompressionLevel` equals to `5`.

https://github.com/elastic/cloud-on-k8s/blob/8f3756d47a5828b0b5525e4ac8519a6729713131/operators/test/e2e/apm/configuration_test.go#L93

This is a small regression introduced when the default APM settings have been removed: 845f8ac.

--- a/operators/test/e2e/apm/configuration_test.go
+++ b/operators/test/e2e/apm/configuration_test.go
@@ -11,6 +11,7 @@ import (
 	apmtype "github.com/elastic/cloud-on-k8s/operators/pkg/apis/apm/v1alpha1"
 	"github.com/elastic/cloud-on-k8s/operators/pkg/apis/common/v1alpha1"
 	commonv1alpha1 "github.com/elastic/cloud-on-k8s/operators/pkg/apis/common/v1alpha1"
+	"github.com/elastic/cloud-on-k8s/operators/pkg/controller/elasticsearch/services"
 	"github.com/elastic/cloud-on-k8s/operators/pkg/utils/k8s"
 	"github.com/elastic/cloud-on-k8s/operators/test/e2e/test"
 	"github.com/elastic/cloud-on-k8s/operators/test/e2e/test/apmserver"
@@ -33,7 +34,8 @@ var APMKeystoreCmd = []string{APMKeystoreBin, APMKeystoreOption}
 type PartialApmConfiguration struct {
 	Output struct {
 		Elasticsearch struct {
-			CompressionLevel int `yaml:"compression_level"`
+			Hosts            []string `yaml:"hosts"`
+			CompressionLevel int      `yaml:"compression_level"`
 		} `yaml:"elasticsearch"`
 	} `yaml:"output"`
 }
@@ -90,7 +92,9 @@ func TestUpdateConfiguration(t *testing.T) {
 				Test: func(t *testing.T) {
 					config, err := partialAPMConfiguration(k, name)
 					require.NoError(t, err)
-					require.Equal(t, config.Output.Elasticsearch.CompressionLevel, 5) // 5 is the expected default value
+					esHost := services.ExternalServiceURL(esBuilder.Elasticsearch)
+					require.Equal(t, config.Output.Elasticsearch.Hosts[0], esHost)
+					require.Equal(t, config.Output.Elasticsearch.CompressionLevel, 0) // CompressionLevel is not set by default
 				},
 			},
 			test.Step{

